Dockerfile

```diff
@@ -7,12 +7,17 @@ COPY documentation/examples/prometheus.yml  /etc/prometheus/prometheus.yml
 COPY console_libraries/                     /usr/share/prometheus/console_libraries/
 COPY consoles/                              /usr/share/prometheus/consoles/
 
-RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
+RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/ && \
+    addgroup -g 1000 prometheus && \
+    adduser -s /bin/sh -S -H -u 1000 -G prometheus prometheus && \
```

Comment:
The reason for not using `nobody` user is that busybox has set the `nobody` to
 `# id nobody
uid=99(nobody) gid=99(nogroup) groups=99(nogroup)`

which is not the same one as ubuntu, or alpine linux which is set to 
`# id nobody
uid=65534(nobody) gid=65534(nogroup) groups=65534(nogroup)`

@jimmidyson i tried to reply inline but the system would not allow me,  yes it is a braking change for users that are using a mounting volume.  However I think the additional security for users that do not assign a uid/user when they run their containers is worth the change.  

if you think this change breaks many things please close the PR, or let me know what changes you want and I will make the necessary changes.